javascripts/previewmode.js

```diff
@@ -28,18 +50,21 @@ function renderPreviewFrame(theContent)
     if (!previewFrame) {
         // might already exist when embedding multiple containers
         previewFrame = createDebugFrame();
-        document.body.appendChild(previewFrame);
+        var sheet = document.createElement('style');
```

Comment:
I wonder if this can cause issues when those styles are blocked by a content security policy like `Content-Security-Policy: style-src https://example.com/` see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src

Can we simply set the CSS on the iframe directly maybe like 
<img width="835" alt="image" src="https://user-images.githubusercontent.com/273120/163919320-9f6f47b7-f977-49ef-86f4-1d2e700d694b.png">

actually we can ignore it because we're already setting the style attribute meaning if the `style` attribute works, then this should work too. we can ignore it.